tasty-script-exitcode/src/Test/Tasty/HUnit/ScriptExit.hs

```diff
@@ -82,7 +84,14 @@ withPython3venv ::
   IO a
 withPython3venv requirements todo =
   withSystemTempDirectory "virtenv" $ \venvDir ->
-  do let process = proc "python3" ["-m", "venv", venvDir]
+  do mpy3 <- findExecutable "python3"
```

Comment:
This bit of code is presumably here to allow the CI to have different names for the Python executable, and not fail if we guess wrong. Is that right? If so, I think a little explanatory comment to that effect would be nice.

Reply:
Yep, exactly. I added a comment describing that.

Reply:
Great!